### PR TITLE
Button

### PR DESCRIPTION
**Pull request by ttahm3d**
- Added Code examples and CSS for buttons
- Changed the types of buttons
- Changed colors to be more accessible on hover

### CODE REVIEW

File: css/components/buttons.css [hunk 1]
```diff
@@ -1,74 +1,138 @@
...
+.btn:active {
+  transform: scale(1.09);
 }
 
 .btn-rounded {
-	border-radius: 0.5rem;
+  border-radius: 0.5rem;
 }
 
 .link {
```
Comment (tanishq20) [inline css/components/buttons.css:18]: I'm supposing that this link is specific for buttons.
So isn't it will be good to give the class name as btn-link.

Comment (ttahm3d) [reply css/components/buttons.css:18]: it can be used as a link like normal anchor tag

Comment (tanishq20) [reply css/components/buttons.css:18]: Oh okay, so it's like universal class. It can be in utility folder.

Comment (ttahm3d) [reply css/components/buttons.css:18]: i'm not following 7-1 strucutre. May be I will change it link-btn

Comment (tanishq20) [reply css/components/buttons.css:18]: Rest everything is fine.

**Review by shubhamkumar648 — COMMENTED** [review]
it looks perfect to me.  just a little suggestion you could add comments

**Comment by ttahm3d** [comment]
> it looks perfect to me. just a little suggestion you could add comments

Will consider it. Thanks

**Pull request merged**